test/runtests.jl

```diff
@@ -1,6 +1,12 @@
 using Measurements, SpecialFunctions, QuadGK, Calculus
 using Test, LinearAlgebra, Statistics, Unitful, Printf, Aqua
-Aqua.test_all(Measurements)
+
+if !isdefined(Base,:get_extension)
+    Aqua.test_all(Measurements)
+else
+    Aqua.test_all(Measurements,stale_deps = false)
+    Aqua.test_stale_deps(Measurements,ignore = [:RecipesBase,:Requires])
```

Comment:
Style:
```suggestion
    Aqua.test_all(Measurements; stale_deps=false)
    Aqua.test_stale_deps(Measurements; ignore=[:RecipesBase, :Requires])
```
I'm strongly of the opinion that everybody should ***always*** start keyword arguments with `;` if they don't want to fall into traps.